### PR TITLE
fix wrong link

### PR DESCRIPTION
**Pull request by zhaoying818**

### CODE REVIEW

**Comment by voorth** [comment]
Hmm... the "absolute" path has worked flawlessly for the last 9 years - are you sure this is needed?

**Comment by zhaoying818** [comment]
> 
> 
> Hmm... the "absolute" path has worked flawlessly for the last 9 years - are you sure this is needed?

when jenkins url is like xxx.com/aaa/bbb , the latest plugin is link to xxx.com/computer/yyy , and is 404

**Comment by Absh-Day** [comment]
@zhaoying818 I was about to raise the same issue then found this PR.
The maven build fails with Caused by: java.lang.ClassNotFoundException: com.sun.mirror.apt.AnnotationProcessorFactory
It seems issue with java8, should work with older versions.

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>built-on-column</artifactId>
-  <version>1.2-SNAPSHOT</version>
```
Comment (qBTp) [inline pom.xml:11]: Keep SNAPSHOT in.

**Review by qBTp — CHANGES_REQUESTED** [review]

**Comment by zhaoying818** [comment]
> 
> 
> @zhaoying818 I was about to raise the same issue then found this PR.
> The maven build fails with Caused by: java.lang.ClassNotFoundException: com.sun.mirror.apt.AnnotationProcessorFactory
> It seems issue with java8, should work with older versions.

I did some change. You can try it.

**Comment by zhaoying818** [comment]
@olivergondza

**Comment by MarkEWaite** [comment]
Thanks for the contribution.  Most crucial changes have been merged in #4 , #6 .  Unwilling to change the package of the plugin because that seems unnecessary.

**Pull request closed without merging**